File: cases/taint/javascript/infeasible-branch-positive/InfeasibleBranchPositive.js
function dfb_source() { // DFB-SOURCE: infeasible-branch-input
  return "tainted";
}

function dfb_sink(value) {} // DFB-SINK: infeasible-branch-sink

function run() {
  let value = "clean";
  if (true) {
    value = dfb_source(); // DFB-WITNESS: feasible-tainted-branch
  }
  dfb_sink(value);
}

File: cases/taint/cpp/infeasible-branch-positive/infeasible_branch_positive.cpp
int dfb_source() { // DFB-SOURCE: infeasible-branch-input
    return 1;
}

void dfb_sink(int value) {} // DFB-SINK: infeasible-branch-sink

void run() {
    int value = 0;
    if (true) {
        value = dfb_source(); // DFB-WITNESS: feasible-tainted-branch
    }
    dfb_sink(value);
}
